src/lib/src/image/image-loader.component.ts

```diff
@@ -29,8 +29,10 @@ export class ImageLoaderComponent implements OnChanges {
   setPlaceholder(isError: boolean) {
     if (isError) {
       this.element.nativeElement.classList.add('placeholder-error');
+      this.element.nativeElement.style['background-image'] = '';
     } else {
       this.element.nativeElement.classList.add('placeholder');
+      this.element.nativeElement.style['background-image'] = '';
```

Comment:
and style doesn't override the background-image in `.placeholder` classes? Seems like we should try to avoid mixing class and style, but if this works... why change it? ;)